.buildkite/gpu_pipeline/pipeline.yml

```diff
@@ -50,3 +50,14 @@ steps:
         agents:
           slurm_gpus: 1
           slurm_time: 23:00:00
+
+      - label: "target_gpu_implicit_baroclinic_wave_4process"
+        command: >
+          julia --project -e 'using CUDA; CUDA.versioninfo()'
+          
+          nsys profile --trace=nvtx,cuda --output=target_gpu_implicit_baroclinic_wave_4process/report julia --color=yes --project=examples examples/hybrid/driver.jl
+          --config_file ${GPU_CONFIG_PATH}target_gpu_implicit_baroclinic_wave_4process.yml
```

Comment:
I think you need an `srun` in there somewhere?

Reply:
The config file also contains the `job_id`, which is used as  a directory name for storing output!